getmyancestors/classes/gedcom.py

```diff
@@ -0,0 +1,315 @@
+# mergemyancestors classes
+from getmyancestors.classes.tree import Indi, Fact, Fam, Memorie, Name, Note, Source
+from getmyancestors.classes.constants import FACT_TYPES
+
```

Comment:
A newline is missing here

Reply:
Fixed. Also found that class Ordinance and constant ORDINANCES weren't imported in that module.